frontend/views/containers/access/LoginForm.vue

```diff
@@ -104,7 +112,13 @@ export default ({
     form: {
       username: {
         [L('A username is required.')]: required,
-        [L('A username cannot contain whitespace.')]: noWhitespace
+        [L('A username cannot contain whitespace.')]: noWhitespace,
+        [L('A username can only contain letters, digits, hyphens or underscores.')]: allowedUsernameCharacters,
+        [L('A username cannot exceed {maxChars} characters.', { maxChars: usernameMaxChars })]: maxLength(usernameMaxChars),
+        [L('A username cannot contain uppercase letters.')]: noUppercase,
+        [L('A username cannot start or end with a hyphen.')]: noLeadingOrTrailingHyphen,
+        [L('A username cannot start or end with an underscore.')]: noLeadingOrTrailingUnderscore,
+        [L('A username cannot contain two consecutive hyphens or underscores.')]: noConsecutiveHyphensOrUnderscores
```

Comment:
Copying these validations does not address #1525. Although the video @leihla uploaded might have shown the problem with the Login Form, that's not the issue, as the original problem happened on the SignUp form. The problem was that Leihla's LastPass extension was filling out the fields in a way that was not triggering the validations at all.

Reply:
Just reverted.